.travis/install.sh

```diff
@@ -63,7 +63,6 @@ images:
       pulpcore: pulpcore
       plugins:
         - ./galaxy_ng
-        - ./galaxy_ng/bindings/galaxy-pulp
```

Comment:
you have to remove it couple lines below also
https://github.com/ansible/galaxy_ng/blob/master/.travis/install.sh#L79

Reply:
missed that. now it's done.